docs/medical-api/handling-data/schema/diagnostic-report.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Diagnostic Report"
+description: "Schema documentation for the Diagnostic Report table in the Metriport data model"
+group: "Schema"
+---
+
+The diagnostic report table contains information about diagnostic reports, including lab results, imaging studies, and other diagnostic findings.
+
+**Reference Table:**
+The `diagnostic_report_references` table contains normalized references from the diagnostic report to other entities. It supports the following reference properties:
+* **subject**: Links to the patient who is the subject of the diagnostic report
+* **encounter**: Links to the encounter where the diagnostic report was generated
+* **performer**: Links to practitioners who performed the diagnostic procedure
+* **result**: Links to observations or other diagnostic results referenced in the report
+
+For detailed information about reference table structure and indexing, see the [Reference Tables documentation](/medical-api/handling-data/schema/reference-tables).
+
+**Presented Forms Table:**
+The `diagnostic_report_presented_forms` table contains the actual content of diagnostic reports, including text, images, and other media. It supports the following properties:
```

Comment:
This likely warrants a diagram, diff from other tables that are just the table and the references table.